mix.exs

```diff
@@ -16,7 +16,8 @@ defmodule Grizzly.MixProject do
       description: description(),
       package: package(),
       docs: docs(),
-      preferred_cli_env: [docs: :docs, "hex.publish": :docs]
+      preferred_cli_env: [docs: :docs, "hex.publish": :docs],
+      xref: [exclude: EEx]
```

Comment:
It might be kind of tacky, but you _could_ probably use:

```elixir
Module.defines?(Mix.Tasks.Compile.Elixir, {:xref_exclude_opts, 2}, :defp)
```
to decide whether or not to include the `[:xref][:exclude]` opts.

See: https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/compile.elixir.ex#L128

It didn't exist < v1.10 😇

Reply:
Thank you for this!

I think I am leaning towards dropping official support for 1.8 and 1.9. The code still should work with those versions as of right now but 1.8 does not support releases, so our primary use case wont ever use 1.8 as Nerves uses mix releases, and 1.9 to 1.10 is rather minor.